installation-bundle/src/Database/Version480Update.php

```diff
@@ -23,13 +24,21 @@ public function shouldBeRun(): bool
     {
         $schemaManager = $this->connection->getSchemaManager();
 
-        if (!$schemaManager->tablesExist(['tl_layout'])) {
+        if (!$schemaManager->tablesExist(['tl_layout', 'tl_files'])) {
             return false;
         }
 
-        $columns = $schemaManager->listTableColumns('tl_layout');
+        $layoutColumns = $schemaManager->listTableColumns('tl_layout');
+        $filesColumns = $schemaManager->listTableColumns('tl_files');
 
-        return isset($columns['picturefill']);
+        if (!isset($filesColumns['importantpartx'])) {
+            return false;
+        }
+
+        return
+            isset($layoutColumns['picturefill'])
+            && 'integer' === $filesColumns['importantpartx']->getType()->getName()
```

Comment:
A second check is actually not needed – at least we use only one check in every other version update class.

Reply:
Should I remove the second check then?

Reply:
Unless there is a specific reason to keep them, yes please.

Reply:
b3ac722f3b9838f5b8b63c80a102a683be93eb23